collect_app/src/main/java/org/odk/collect/android/activities/GeoPointMapActivity.java

```diff
@@ -219,89 +213,67 @@ public void onClick(View v) {
         });
 
         reloadLocation.setEnabled(false);
-        reloadLocation.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if (marker != null) {
-                    marker.remove();
-                }
-                latLng = null;
-                marker = null;
-                setClear = false;
-                latLng = new LatLng(location.getLatitude(), location.getLongitude());
-                markerOptions.position(latLng);
-                if (marker == null) {
-                    marker = map.addMarker(markerOptions);
-                    if (draggable && !readOnly) {
-                        marker.setDraggable(true);
-                    }
+        reloadLocation.setOnClickListener(v -> {
```

Comment:
I have a question, for Collect code, should we use lambda for all those methods? I found lots of lambda expressions but there are still many methods not be refactored with that. Do you think we have the necessary to keep the consistency? We can open a new issue maybe.

Reply:
I honestly think we should refactor as we touch code that have these listeners. Most of it is boilerplate so the lambda makes things more concise. Also the codebase will always have a mixture of lambda and normal expressions so it should be fine 😊

Reply:
👍 Great, @jd-alexander . Can you help to do this refactor? You can see #2294